sw/device/tests/penetrationtests/json/otp_fi_commands.h

```diff
@@ -0,0 +1,59 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_TESTS_PENETRATIONTESTS_JSON_OTP_FI_COMMANDS_H_
+#define OPENTITAN_SW_DEVICE_TESTS_PENETRATIONTESTS_JSON_OTP_FI_COMMANDS_H_
+#include "sw/device/lib/ujson/ujson_derive.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// clang-format off
+
+#define OTPFI_SUBCOMMAND(_, value) \
+    value(_, Init) \
+    value(_, VendorTest) \
+    value(_, OwnerSwCfg) \
+    value(_, HwCfg) \
+    value(_, LifeCycle)
+UJSON_SERDE_ENUM(OtpFiSubcommand, otp_fi_subcommand_t, OTPFI_SUBCOMMAND);
+
+#define OTPFI_VENDORTEST_PARTITION(field, string) \
+    field(vendor_test_comp, uint32_t, 16) \
+    field(vendor_test_fi, uint32_t, 16) \
+    field(otp_status_codes, uint32_t) \
+    field(otp_error_causes, uint8_t, 10) \
+    field(alerts, uint32_t, 3)
+UJSON_SERDE_STRUCT(OtpFiVendortestPartition, otp_fi_vendortest_partition_t, OTPFI_VENDORTEST_PARTITION);
+
+#define OTPFI_OWNERSWCFG_PARTITION(field, string) \
+    field(owner_sw_cfg_comp, uint32_t, 200) \
```

Comment:
Where are these sizes coming from (e.g. 200)? Would it make sense to use variables here to make sure that we do not accidentally try sending too many / less data over UART back to the host?

Reply:
That would be good, thanks!